model-sync-gradle-test/ci.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+(
```

Comment:
Is there a specific reason for this subshell being started here? Everything is included, so couldn't that also just be the outer script shell instance?